2019-03-24 14:29:30.410654: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-03-24 14:29:30.554785: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1105] Found device 0 with properties: 
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.582
pciBusID: 0000:03:00.0
totalMemory: 10.92GiB freeMemory: 10.76GiB
2019-03-24 14:29:30.554884: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1195] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:03:00.0, compute capability: 6.1)
2019-03-24 14:30:21.729399: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1195] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:03:00.0, compute capability: 6.1)
Traceback (most recent call last):
  File "../exps/tter_test.py", line 147, in <module>
    save_to_file(result_dicts, result_config['path'])
  File "../src/data/t_metrics.py", line 21, in save_to_file
    np.save(path + '_' + process_key + '_' + metric_key, metrics[metric_key])
  File "/clusterFS/home/student/kopp13/miniconda3/envs/tensorflow/lib/python3.5/site-packages/numpy/lib/npyio.py", line 492, in save
    fid = open(file, "wb")
FileNotFoundError: [Errno 2] No such file or directory: '../nr/exp_8_va_b_elogl.npy'
